internal/batch/jobs/jobs.go

```diff
@@ -183,6 +195,10 @@ func get(ctx context.Context, id string) error {
 	return sendRequestAndPrintBatchJob(ctx, http.MethodGet, fmt.Sprintf("%s/%s", path, id), &jv1.GetBatchJobRequest{})
 }
 
+func delete(ctx context.Context, id string) error {
+	return sendRequestAndPrintBatchJob(ctx, http.MethodDelete, fmt.Sprintf("%s/%s", path, id), &jv1.GetBatchJobRequest{})
```

Comment:
Would be nice to have a confirmation prompt to follow the existing code.

https://github.com/llm-operator/cli/blob/main/internal/workspaces/notebooks/notebooks.go#L231